ngraph_onnx/onnx_importer/utils/pool.py

```diff
@@ -72,30 +71,36 @@ def reduce_extra_dims(spatial_dims_count, param_shape, onnx_node):
     return param_shape
 
 
-def make_pooling_op(onnx_node, ng_inputs, custom_pool_params=None):
-    # type: (NodeWrapper, List[NgraphNode], Dict) -> NgraphNode
+def make_pooling_op(onnx_node, ng_inputs, is_global=False, custom_pool_params=None):
```

Comment:
Please make function `make_global_pooling_op`
In that function calculate the `kernel_shape`
Pass `kernel_shape` to `make_pooling_op`.
Perhaps pass it in `custom_pool_params`. What are `custom_pool_params` used for currently? 
Please document `custom_pool_params` or remove.